bindings/native/go/dist/stronghold_native.h

```diff
@@ -0,0 +1,44 @@
+#include <stdarg.h>
```

Comment:
This file looks like it's being generated by `bindings/native/bindgen.sh` Please do not check in generated sources.

Reply:
Unfortunately, we need the header files for the Go lib. (Look at native.go CGO annotations)
As long as we have no build pipeline which combines things (cargo build, bindgen, zip) I would like to keep it for now.